libs/application/templates/financial-aid/src/fields/Summary/SummaryForm.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useState, useMemo } from 'react'
+import { useIntl } from 'react-intl'
+import cn from 'classnames'
+
+import { Text, Box } from '@island.is/island-ui/core'
+import {
+  Employment,
+  getNextPeriod,
+  HomeCircumstances,
+  estimatedBreakDown,
+  aidCalculator,
+  martialStatusTypeFromMartialCode,
+  MartialStatusType,
+} from '@island.is/financial-aid/shared/lib'
+
+import * as m from '../../lib/messages'
+import {
+  ApproveOptions,
+  FAFieldBaseProps,
+  SummaryComment,
+} from '../../lib/types'
+import { Routes } from '../../lib/constants'
+import { DescriptionText, Breakdown } from '../index'
+import {
+  formatAddress,
+  formatBankInfo,
+  getMessageApproveOptions,
+  getMessageApproveOptionsForIncome,
+  getMessageEmploymentStatus,
+  getMessageHomeCircumstances,
+  getMessageFamilyStatus,
+} from '../../lib/formatters'
+import { findFamilyStatus } from '../../lib/utils'
+import useApplication from '../../lib/hooks/useApplication'
+import * as styles from '../Shared.css'
+import { hasSpouse } from '../../lib/utils'
+import FormInfo from './FormInfo'
+import ContactInfo from './ContactInfo'
+import UserInfo from './UserInfo'
+import Comment from './Comment'
+import Files from './Files'
```

Comment:
maybe create index in this file where everything is imported in summary, this is a bit cluttered?